livemaps-api/APIFunctions/DeviceState.cs

```diff
@@ -38,17 +36,10 @@ public static class DeviceState
 
                 fileName = fileName.ToLower();
                
-                var devicestateref = container.GetBlockBlobReference(fileName);
-                using (var ms = new MemoryStream())
-                {
-                    await devicestateref.DownloadToStreamAsync(ms);
-                    ms.Position = 0;
-                    using (StreamReader reader = new StreamReader(ms, Encoding.UTF8))
-                    {
-                        var deviceStateData = reader.ReadToEnd();
-                        return new OkObjectResult(deviceStateData);
-                    }
-                };
+                var devicestateref = container.GetBlobClient(fileName);
```

Comment:
The naming of variables doesn't follow C# norms;  as an official sample, we should update these.   For example, this should be `deviceStateRef`  or `deviceState`  or `deviceStateReference`.   

Leaving one comment here, but this pops up across the code for this sample.